extensions/dash/src/hooks/useDashApp.ts

```diff
@@ -1,20 +1,22 @@
 import { useEffect, useState } from "react";
 import { Application, getApplications } from "@raycast/api";
 
-export default function useDashApp() {
+export default function useDashApp(): [Application | null, boolean] {
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
a good use case for `usePromise` 😄

Reply:
yeah I meant to switch to "migrating" to `@raycast/utils`, just thought it'd be better to do that as a separate step 😸